### PR TITLE
Update oracle-connection-type-ssrs.md

### PR DESCRIPTION
**Pull request by kanecoMSFT**
Added steps to configure ODP.NET Unmanaged Client Libraries which are required for Power BI Reports since Power BI Report Server September 2019 Release

### CODE REVIEW

**Comment by PRMerger10** [comment]
@kaneconway : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 1]
```diff
@@ -25,6 +25,14 @@ Power BI Report Server and SQL Server Reporting Services 2016 and 2017 all use M
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
### Power BI reports use Unmanaged ODP.NET
```

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 2]
```diff
@@ -25,6 +25,14 @@ Power BI Report Server and SQL Server Reporting Services 2016 and 2017 all use M
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]  
+ > Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
 
 Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
```

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 3]
```diff
@@ -25,6 +25,14 @@ Power BI Report Server and SQL Server Reporting Services 2016 and 2017 all use M
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]  
+ > Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
+ > 1. Register ODP.NET Unmanaged Client to GAC:
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
 1. Register ODP.NET Unmanaged Client to GAC:
```

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 4]
```diff
@@ -25,6 +25,14 @@ Power BI Report Server and SQL Server Reporting Services 2016 and 2017 all use M
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]  
+ > Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
+ > 1. Register ODP.NET Unmanaged Client to GAC:
+ >     C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:gac /providerpath:C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\Oracle.DataAccess.dll
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
    C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:gac /providerpath:C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\Oracle.DataAccess.dll
```

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 5]
```diff
@@ -25,6 +25,14 @@ Power BI Report Server and SQL Server Reporting Services 2016 and 2017 all use M
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]  
+ > Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
+ > 1. Register ODP.NET Unmanaged Client to GAC:
+ >     C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:gac /providerpath:C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\Oracle.DataAccess.dll
+ > 2. Add ODP.NET Unmanaged Client entries to machine.config:
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
 2. Add ODP.NET Unmanaged Client entries to machine.config:
```

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 6]
```diff
@@ -25,6 +25,14 @@ Power BI Report Server and SQL Server Reporting Services 2016 and 2017 all use M
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]  
+ > Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
+ > 1. Register ODP.NET Unmanaged Client to GAC:
+ >     C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:gac /providerpath:C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\Oracle.DataAccess.dll
+ > 2. Add ODP.NET Unmanaged Client entries to machine.config:
+ >     C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odp /frameworkversion:v4.0.30319 /productversion:4.122.18.3
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
    C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odp /frameworkversion:v4.0.30319 /productversion:4.122.18.3
```

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 7]
```diff
@@ -25,6 +25,14 @@ Power BI Report Server and SQL Server Reporting Services 2016 and 2017 all use M
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]  
+ > Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
+ > 1. Register ODP.NET Unmanaged Client to GAC:
+ >     C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:gac /providerpath:C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\Oracle.DataAccess.dll
+ > 2. Add ODP.NET Unmanaged Client entries to machine.config:
+ >     C:\oracle64\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odp /frameworkversion:v4.0.30319 /productversion:4.122.18.3
+ >
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
```

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 8]
```diff
@@ -34,6 +42,14 @@ The following steps are only needed for when using the latest 18x drivers. They
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
### Power BI reports use Unmanaged ODP.NET
```

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 9]
```diff
@@ -34,6 +42,14 @@ The following steps are only needed for when using the latest 18x drivers. They
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]  
+ > Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
```

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 10]
```diff
@@ -34,6 +42,14 @@ The following steps are only needed for when using the latest 18x drivers. They
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]  
+ > Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
+ > 1. Register ODP.NET Unmanaged Client to GAC:
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
1. Register ODP.NET Unmanaged Client to GAC:
```

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 11]
```diff
@@ -34,6 +42,14 @@ The following steps are only needed for when using the latest 18x drivers. They
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]  
+ > Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
+ > 1. Register ODP.NET Unmanaged Client to GAC:
+ >     C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:gac /providerpath:C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\Oracle.DataAccess.dll
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
    C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:gac /providerpath:C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\Oracle.DataAccess.dll
```

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 12]
```diff
@@ -34,6 +42,14 @@ The following steps are only needed for when using the latest 18x drivers. They
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]  
+ > Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
+ > 1. Register ODP.NET Unmanaged Client to GAC:
+ >     C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:gac /providerpath:C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\Oracle.DataAccess.dll
+ > 2. Add ODP.NET Unmanaged Client entries to machine.config:
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
2. Add ODP.NET Unmanaged Client entries to machine.config:
```

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 13]
```diff
@@ -34,6 +42,14 @@ The following steps are only needed for when using the latest 18x drivers. They
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]  
+ > Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
+ > 1. Register ODP.NET Unmanaged Client to GAC:
+ >     C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:gac /providerpath:C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\Oracle.DataAccess.dll
+ > 2. Add ODP.NET Unmanaged Client entries to machine.config:
+ >     C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odp /frameworkversion:v4.0.30319 /productversion:4.122.18.3
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
    C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odp /frameworkversion:v4.0.30319 /productversion:4.122.18.3
```

File: docs/reporting-services/report-data/oracle-connection-type-ssrs.md [hunk 14]
```diff
@@ -34,6 +42,14 @@ The following steps are only needed for when using the latest 18x drivers. They
 3. Add ODP.NET Managed Client entries to machine.config:
     C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odpm /frameworkversion:v4.0.30319 /productversion:4.122.18.3
 
+ > [!NOTE]  
+ > Power BI Reports use **Unmanaged ODP.NET**. Follow these steps to register Unmanaged ODP.NET:
+ > 1. Register ODP.NET Unmanaged Client to GAC:
+ >     C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:gac /providerpath:C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\Oracle.DataAccess.dll
+ > 2. Add ODP.NET Unmanaged Client entries to machine.config:
+ >     C:\oracle32\product\18.0.0\client_1\odp.net\bin\4\OraProvCfg.exe /action:config /force /product:odp /frameworkversion:v4.0.30319 /productversion:4.122.18.3
+ >
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/oracle-connection-type-ssrs.md]:
```suggestion
```

**Comment by maggiesMSFT** [comment]
#sign-off
Hi, @kanecoMSFT -- thanks for these changes! I made them into H3s instead of Notes. Notes typically don't have numbered steps in them. 
Also, I'm going to explain to you offline how to make your changes in the Power BI private repo instead of the public one in the future.

**Comment by PRMerger13** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@maggiesMSFT I wasn't able to "commit" your suggested changes, so I applied them by using the pencil edit option. Merging now.

**Comment by maggiesMSFT** [comment]
@ktoliver thanks so much! Was it something about the way I had done the changes?

**Pull request merged**